components/board/fake/board.go

```diff
@@ -12,11 +12,13 @@ import (
 	"github.com/pkg/errors"
 	"go.uber.org/multierr"
 	pb "go.viam.com/api/component/board/v1"
+	"go.viam.com/utils"
 
 	"go.viam.com/rdk/components/board"
 	"go.viam.com/rdk/grpc"
 	"go.viam.com/rdk/logging"
 	"go.viam.com/rdk/resource"
+	rdkutils "go.viam.com/rdk/utils"
```

Comment:
[non-actionable comment just to give background info]

Sorry you need to import this separate from `go.viam.com/utils`. When the kinks have been worked out, `StoppableWorkers` will be moved over to there, and you won't have a name conflict with this import. but that hasn't happened yet.